test/index.test.ts

```diff
@@ -17,6 +17,13 @@ describe('lib/index.ts', () => {
     const element = await queries.getByText(document, 'Hello h1')
     expect(await queries.getNodeText(element)).toEqual('Hello h1')
   })
+  
+  it('should support custom data-testid names', async () => {
+    configure({testIdAttribute: 'data-id'})
```

Comment:
let's add another test to make sure users can call configure twice (would currently be broken)